### ISSUE TITLE
Solutions in the same directory should be able to define different bindings

### ISSUE THREAD

**Issue by fmallet** [SonarSource/sonarlint-visualstudio#190]
If two Visual Studio Solutions are stored in the same directory, then they will de-facto share the same binding information

**Reproducing**:
- create solution _MyApp1_ in _path/mydir_
- create solution _MyApp2_ in _path/mydir_
- if _MyApp1/2_ were stored in distinct sub-directories then copy-paste the contents of those sub-directories in _path/mydir_
- open _MyApp1_ and bind it to a SonarQube project
- open _MyApp2_ : it is bound to the same SonarQube project. To bind it to another project you must first unbind it, which will unbind _MyApp1_
- bottom line: _MyApp1_ and _MyApp2_ share the same binding

**Root-cause analysis**:
Binding information is stored next to the solution in a _SonarQube_ folder (see attached screenshot). Multiple Solutions in the same directory therefore share the same _SonarQube_ folder/binding.
<img width="400" alt="multiple_solutions" src="https://cloud.githubusercontent.com/assets/548729/26193898/e7baa992-3bb6-11e7-88bb-3749c4ed51ed.png">

Comes from [SVS-113](https://jira.sonarsource.com/browse/SVS-113)
